src/arrayMethodSort.js

```diff
@@ -3,10 +3,33 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const compare = compareFunction || function(a, b) {
+      const strA = String(a);
+      const strB = String(b);
+
+      if (strA > strB) {
+        return 1;
+      }
+
+      if (strA < strB) {
+        return -1;
+      }
+
+      return 0;
+    };
+
+    for (let prev = 0; prev < this.length; prev++) {
+      for (let next = prev + 1; next < this.length; next++) {
```

Comment:
Consider using more descriptive variable names such as 'nextIndex' instead of 'next'.